react-with-graphql/src/Components/TodoList/TodoList.js

```diff
@@ -1,46 +1,44 @@
 import React from 'react';
 import { Query, Mutation } from "react-apollo";
 
-import { GET_ALLTODOS, REMOVE_TODO, UPDATE_TODO, SUBSCRIBE_NEWTODO } from '../../queries';
+import { GET_ALLTODOS, REMOVE_TODO, UPDATE_TODO,SUBSCRIBE_NEWTODO } from '../../queries';
 
 import './TodoList.css';
 
 
-const updateCache = (cache, { data: { deletetodo } }) => {
+const updateCache = (cache, { data: { deleteTodo } }) => {
     cache.writeQuery({
         query: GET_ALLTODOS,
-        data: { todos: deletetodo }
+        data: { todos: deleteTodo }
     });
 }
 
 
-
 const TodoList = () => (
     <Query
         query={GET_ALLTODOS}
     >
-        {({ loading, error, data, subscribeToMore }) => {
+        {({ loading, error, data,subscribeToMore }) => {
             if (loading) return <p>Loading...</p>;
             if (error) return <p>Error :(</p>;
 
-
             return (
                 <div className="completed-list">
                     <div className="todo-pending">
                         {data.todos.filter(todo => todo.complete === false).map(function (todo, index) {
                             return (<div className="todo-item" key={index}>
-                                
                                 <Mutation mutation={UPDATE_TODO}>
-                                    {(updatetodo) => (
+                                    {(updateTodo) => (
                                         <div className="div-checkbox">
-                                            <input type="checkbox" className="complete-checkbox" checked={false} onClick={() =>{updatetodo({ variables:{todoid:todo._id}})}} />
+                                            {console.log(todo.id)}
```

Comment:
i don't want to see this in a pr again. I told you multiple times that pr shouldn't contain logs like this